src/context.ts

```diff
@@ -26,7 +26,7 @@ export interface OperonContext {
   readonly span: Span;
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  getConfig(key: string): any;
+  getConfig<T>(key: string): T;
```

Comment:
Did we drop the idea of providing a default value? 

If there's a default value, TS can infer getConfig type parameter. So something like `ctx.getConfig('answer', 42)` without needing to specify `<number>`

Reply:
I just added support for the default value. I did update the tests with `<type>` specified, but I agree it is not needed, TSC can infer the type. Specifying the type just helps speed up compilation